src/components/pagination/Pagination.tsx

```diff
@@ -0,0 +1,114 @@
+import React, { ReactNode } from 'react';
+import Box from '../box';
+import { TextBody } from '../typography';
+import { IconChevronLeftMediumOutline, IconChevronRightMediumOutline } from '@teamleader/ui-icons';
+import cx from 'classnames';
+import theme from './theme.css';
+import { BoxProps } from '../box/Box';
+import { GenericComponent } from '../../@types/types';
+
+type ChildrenFunctionArgument = {
```

Comment:
interface

Reply:
Rule of thumb: Always use `interface` unless you need `type`.

Reply:
Yeah, already read this, but don't know how I missed this. Thank you guys!